chia/rpc/util.py

```diff
@@ -2,18 +2,56 @@
 
 import logging
 import traceback
-from typing import Any, Callable, Coroutine, Dict, List, Optional, Tuple
+from dataclasses import dataclass
+from typing import Any, Callable, Coroutine, Dict, List, Optional, Tuple, Type, TypeVar, get_type_hints
 
 import aiohttp
+from typing_extensions import TypedDict, dataclass_transform
 
 from chia.types.blockchain_format.coin import Coin
 from chia.util.json_util import obj_to_response
+from chia.util.streamable import Streamable, streamable
 from chia.wallet.conditions import Condition, ConditionValidTimes, conditions_from_json_dicts, parse_timelock_info
 from chia.wallet.transaction_record import TransactionRecord
 from chia.wallet.util.tx_config import TXConfig, TXConfigLoader
 
 log = logging.getLogger(__name__)
 
+RpcEndpoint = Callable[..., Coroutine[Any, Any, Dict[str, Any]]]
+MarshallableRpcEndpoint = Callable[..., Coroutine[Any, Any, Streamable]]
+
+
+class RequestType(TypedDict):
+    pass
+
+
+_T_Streamable = TypeVar("_T_Streamable", bound="Streamable")
```

Comment:
```suggestion
_T_Streamable = TypeVar("_T_Streamable", bound=Streamable)
```

Reply:
Deleted this, I wasn't using it correctly